bigtable/cmd/cbt/cbt.go

```diff
@@ -581,7 +607,7 @@ func doCount(ctx context.Context, args ...string) {
 	if len(args) != 1 {
 		log.Fatal("usage: cbt count <table>")
 	}
-	tbl := getClient(bigtable.ClientConfig{}).Open(args[0])
+	tbl := getTable(bigtable.ClientConfig{}, args[0])
```

Comment:
I am having some difficulty fully understanding the need of this refactoring and its relation to the timeout change. Can you describe it a bit?

Reply:
I wanted to test that a proper context (with  a timeout) was passed to `ReadRows`.  

`getTable` returns a `tableLike` object, which simply has `ReadRows`. (I think I expand it in another PR, IIRC :) )

Normally, `getTable` calls `getClient(clientConf).Open(tableName)`, but if the global `table` is set, it returns that.

`timeout_test.go` sets `table` to a mock that captures the context passed to `ReadRows` so we can assert that it was set correctly.